packages/list/src/renderJson.ts

```diff
@@ -17,8 +17,8 @@ export default async function (
   const jsonArr = await Promise.all(pkgs.map(async (pkg) => {
     const jsonObj = {
       name: pkg.name,
-
       version: pkg.version,
+      workspace: pkg.path,
```

Comment:
Unlike Yarn, we currently call the monorepo a workspace.

Probably better to name it path, location, or dir.

Reply:
okay,  sounds good to me